Source/DafnyCore/DafnyOptions.cs

```diff
@@ -1332,9 +1332,9 @@ the program.
 
 /extractCounterexample
     If verification fails, report a detailed counterexample for the
-    first failing assertion. Requires specifying the /mv option as well
-    as /proverOpt:O:model_compress=false and
-    /proverOpt:O:model.completion=true.
+    first failing assertion. Requires specifying the /mv:<file> option as well
+    as /proverOpt:O:model_compress=false (or /proverOpt:O:model.compact=false if
+    your version of z3 is >= 4.8.7) and /proverOpt:O:model.completion=true.
```

Comment:
We'll probably want to swap what's in parentheses once we adopt a newer version by default.

Reply:
Or maybe update this text to match what's in the message in `DafnyModel.cs`, which is more symmetric?